frontend/src/components/HokieBirdColor.tsx

```diff
@@ -52,7 +54,9 @@ export function HokieBirdColoring({
     left_foot: "",
     right_foot: "",
   });
-
+  const { user } = useAuth();
+  const startTime = new Date().getTime();
+  let answer = "";
```

Comment:
would try not to have this var. It would have to have be state or try to derive this from other state